extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -473,7 +473,7 @@ export class VisibilityManagerForDoc extends VisibilityManager {
     };
     this.unsubscribe(this.viewport_.onScroll(ticker));
     this.unsubscribe(this.viewport_.onChanged(ticker));
-    setTimeout(ticker);
+    ticker();
```

Comment:
I liked `setTimeout` because that's how native API works. The pings are ALWAYS asynchronous. So, this was not accidental :)